src/OrbitBase/include/OrbitBase/Append.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <vector>
+
+#ifndef ORBIT_BASE_APPEND_H_
+#define ORBIT_BASE_APPEND_H_
+
+namespace orbit_base {
+
+template <class T>
+inline void Append(std::vector<T>& dest, const std::vector<T>& source) {
```

Comment:
add unit-tests? also probably drop inline - let the compiler figure this out.

Reply:
Isn't inline needed because the definition is in the header, or does this not apply to templates?

Reply:
I do not think it does.

Reply:
Which part of my sentence are you referring to?

Reply:
... apply to templates

Reply:
Right. Both done.